http/document_service.go

```diff
@@ -72,30 +80,34 @@ func NewDocumentHandler(b *DocumentBackend) *DocumentHandler {
 }
 
 type documentResponse struct {
-	Links map[string]string `json:"links"`
+	Links         map[string]string `json:"links"`
+	CreatedAt     string            `json:"createAt"`
+	LastUpdatedAt string            `json:"lastUpdatedAt"`
 	*influxdb.Document
 }
 
-func newDocumentResponse(ns string, d *influxdb.Document) *documentResponse {
+func newDocumentResponse(ns string, d *influxdb.Document, createdAt, lastUpdatedAt time.Time) *documentResponse {
```

Comment:
Just an thought, but a pattern I've seen @desa use to avoid having to decorate this information in the response is doing the decoration on the kv layer, using `DocumentDecorator`. This has the added benefit of decoupling KeyValueLog from the Document service.